### PR TITLE
Add safe wrappers around localStorage

### PR DESCRIPTION
**Pull request by mickeyreiss-visor**
@apucacao Here's an attempt at solving #55. I modeled the Store API based on [store.js](https://github.com/marcuswestin/store.js/).

I didn't take the liberty of adding a dependency to your SDK. However, as a consumer, I'd be open to that outcome, especially given the the lightweight and and robust nature of this particular library.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ var Requestor = require('./Requestor');
 var Identity = require('./Identity');
 var utils = require('./utils');
 var messages = require('./messages');
+var store = require('./store');
```
Comment (apucacao) [inline src/index.js]: Does it matter than we require `store.js` here while the module is actually named `Store.js` (capital `S`)?

Comment (mickeyreiss-visor) [reply src/index.js]: Good catch. That may cause problems in some environments. Fixed!

**Comment by mickeyreiss-visor** [comment]
@apucacao pushed a fix and also rebased on `master`.

**Comment by apucacao** [comment]
This should go out shortly. Thanks again @mickeyreiss-visor !

**Pull request merged**